client-mvp-04/src/actions/types.js

```diff
@@ -0,0 +1,3 @@
+// Auth
+
```

Comment:
Remove this line.

Reply:
The Auth comment? I put it there as a way to group the different types into categories, but I can remove it..